scalekit/actions/actions.py

```
@@ -323,6 +324,89 @@ def decorator(func):
+        params = query_params or {}
+
+        proxy_headers = {
+            "connection_name": connection_name,
+            "Connection_name": connection_name,
+            "identifier": identifier,
+        }
+        if headers:
+            proxy_headers.update(headers)
+        req_headers = core.get_headers(proxy_headers)
+
+        response = requests.request(
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)